docs/public/installing-odo.adoc

```diff
@@ -24,6 +24,15 @@ v2.0.0-alpha of odo uses **devfile** as its default deployment mechanism.
 # chmod +x /usr/local/bin/odo
 ----
 
+== Installing odo on Linux on IBM Power
+
+=== Binary installation
+
+----
+# curl -L https://github.com/openshift/odo/releases/download/v2.0.0-alpha-2/odo-linux-ppc64le -o /usr/local/bin/odo
```

Comment:
@sarveshtamba We have GA bits https://github.com/openshift/odo/releases/tag/v2.0.0. So can you please update the references in the doc and commit it into ```master``` branch not ```openshift:alpha```

Reply:
PR for master branch has already been merged here https://github.com/openshift/odo/pull/4049
As per discussion closing this PR # 4055 as alpha branch is no longer used.